src/UnigramWordPredictor.java

```diff
@@ -52,6 +53,26 @@ public void train(Scanner scanner) {
     List<String> trainingWords = tokenizer.tokenize(scanner);
 
     // TODO: Convert the trainingWords into neighborMap here
+    //initialize map (we in this case cannot do this within the 
+    //constructor, so we can point neighborMap within method to
+    //a new HashMap<>()
+    this.neighborMap = new HashMap<>();
+
+    //iterate over tokens; build NeighborMap
+    for (int i = 0; i < trainingWords.size() - 1; i++) {
+      String word = trainingWords.get(i); 
+      String nextWord = trainingWords.get(i + 1);
+
+      //check if word not in map
+      if (!neighborMap.containsKey(word)) {
```

Comment:
I like that you're adding comments, but you can consider leaving out ones that are a bit redundant here and throughout your program. It's already clear what the line does; the comment doesn't add any extra information.